hoot-core/src/main/cpp/hoot/core/conflate/poi-polygon/extractors/PoiPolygonAddressScoreExtractor.h

```diff
@@ -30,24 +30,24 @@
 // hoot
 #include <hoot/core/OsmMap.h>
 #include <hoot/core/conflate/extractors/FeatureExtractorBase.h>
-#include <hoot/core/language/ToEnglishTranslator.h>
 #include <hoot/core/util/Configurable.h>
 #include <hoot/core/algorithms/ExactStringDistance.h>
-
-// Qt
-#include <QMultiMap>
+#include <hoot/core/algorithms/AddressParser.h>
 
 namespace hoot
 {
 
-class PoiPolygonAddress;
+class Address;
 
 /**
  * Calculates the address similarity score of two features involved in POI/Polygon conflation.
- * Only exact string matches yield a positive score.  This can translate addresses, but doesn't
- * handle abbreviations.
+ * Only exact string matches yield a positive score.
  *
- * @todo libaddressinput and libpostal may be able to clean up logic in this class quite a bit
+ * Some effort was spent in validating addresses with libaddressinput
+ * (https://github.com/googlei18n/libaddressinput).  It was found that yields no utility since the
+ * features we are comparing are geographically close, don't need higher level address info
+ * (state, city, etc.), and we basically assume feature address are valid anyway...we're just trying
+ * to match them to each other.
  */
```

Comment:
I found that libaddressinput was really only useful for filling in missing address fields...usually city/state/zip.  Since we're only conflating features that are assumed to fairly close together, adding those fields didn't really buy us anything.